tools/config_gen.py

```diff
@@ -41,6 +41,7 @@
 ]
 
 SKIP_GROUPS = ["api_pecan", "rbac", "results_tracker"]
+SKIP_OPTIONS = ["json_library"]
```

Comment:
Now that `json_library` is not an option, do you still want to leave this here?

Reply:
I agree. We talked about not having user config options for json lib.

Reply:
Yeah, the config was removed but the this flag was left here so we can also support ignoring specific config options (previously this script didn't support that).

I will change ``SKIP_OPTIONS`` to an empty list.